docker-compose.yml

```diff
@@ -468,6 +468,54 @@ services:
     ports:
       - "3170"
 
+  contacts:
+    image: govuk/contacts:${CONTACTS_COMMITISH:-master}
```

Comment:
Ahh I see this seems to have this image name because of the jenkins job, that's a bit annoying since it's called contacts-admin in most other contexts. Not sure how painful to resolve.

Reply:
Yeah, the Jenkins job was the source of confusion our end.